glslc/src/file_includer.h

```diff
@@ -45,6 +49,7 @@ class FileIncluder : public shaderc::CompileOptions::IncluderInterface {
   shaderc_includer_response response_;
   std::vector<char> file_content_;
   std::string file_full_path_;
+  std::set<std::string> file_path_trace_;
```

Comment:
Say what this is.  Does order matter, or not?
I suspect it should be called source_files_used_